src/app/auth/login/login.component.html

```diff
@@ -36,5 +36,12 @@
       </button>
     </form>
   </mat-card-content>
+  <mat-card-content *ngIf="!localAuthEnabled && oidcEnabled">
+    <form>
+      <button *ngIf="oidcEnabled" i18n="@@login.login_oidc" class="form-login-button" color="primary" mat-raised-button (click)="oidcAuthClick()" >
+        Login with Zitadel
```

Comment:
It is not necessarily Zitadel
Should we call it "Login with OIDC" to make it clearer?
And I guess it should be consistent with the text above?